### DISCUSSION TITLE
Dockerfile to run rpi-mqtt-monitor on OpenMower

### DISCUSSION THREAD

**Discussion by 2m** [hjelev/rpi-mqtt-monitor#116] [Show and tell]
Thanks for creating this project! I was looking for something similar to monitor my [OpenMower](https://openmower.de/) running on RPi.

I created a Dockerfile and systemd service config specifically for OpenMower here: https://github.com/2m/rpi-mqtt-monitor-docker

I also hacked a quick [configuration override from environment vars script](https://github.com/2m/rpi-mqtt-monitor-docker/blob/main/config.py) which gives a convenient way to configure this library from the systemd unit file.

This is quite a specific use-case, so not sure if it is worth to merge anything from my repo to here. But let me know if you think otherwise. :)

**Comment by hjelev** [comment]
Hey are you really able to run my script in a docker file and get real readings?  I have tried in the past and failed as it was only getting readings from the container env.

**Reply by 2m** [reply]
Hi! When I initially worked on this, I realized that I want more various metrics for OpenMower. For that I have reimplemented monitoring for OpenMpwer a bit differently here: https://github.com/2m/rpimon

But it still runs inside the docker container. I run it with [--privileged](https://github.com/2m/rpimon/blob/main/rpimon.service#L22) so it is possible to monitor the system as opposed to the inside of the container.
